requirements-dev.txt

```diff
@@ -7,6 +7,7 @@ Babel
 configparser!=3.5.2,!=3.5.3,!=3.7.0
 django-debug-toolbar>=1.9.1,<1.10.0
 flake8>=3.5.0,<3.6.0
+graphviz
```

Comment:
It wouldn't hurt to bind this to a minor version. Same goes for inflect and ipython which I think I added at some point :|